### PR TITLE
[v2.0.8] Issue 3855 fixes

### PR DESCRIPTION
**Pull request by chiranjibi009**
This PR contains the issue fixes for issue #3855

### CODE REVIEW

**Review by moschetti — APPROVED** [review]

**Comment by aswinijena100** [comment]
@googlebot I consent.

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/util/UserManagementUtil.java [hunk 1]
```diff
@@ -171,4 +171,16 @@ public void deleteUserInfoInAuthServer(String userId) {
     restTemplate.exchange(
         appConfig.getAuthServerDeleteStatusUrl(), HttpMethod.DELETE, entity, Void.class, userId);
   }
+
+  public static String getCurrentDate() {
```
[the remark this answers is not in the document]
Comment (aswinijena100) [reply participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/util/UserManagementUtil.java:175]: ignoring Javadoc comment

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/util/UserManagementUtil.java [hunk 2]
```diff
@@ -171,4 +171,16 @@ public void deleteUserInfoInAuthServer(String userId) {
...
+    Calendar currentDate = Calendar.getInstance();
+    SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
+    return formatter.format(currentDate.getTime());
+  }
+
+  public static String getCurrentTime() {
```
[the remark this answers is not in the document]
Comment (aswinijena100) [reply participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/util/UserManagementUtil.java:181]: ignoring Javadoc comment

**Comment by aswinijena100** [comment]
@moschetti This PR required re approval post branch update.

**Review by moschetti — APPROVED** [review]

**Pull request merged**